packages/ketcher-react/src/style/mixins.less

```diff
@@ -532,3 +304,9 @@
   max-height: 80vh;
   overflow: auto;
 }
+
+.kbd() {
```

Comment:
Could you please give this mixin a more meaningful name?

Reply:
it is mixin for `kbd`-elements